hail/python/hailtop/utils/utils.py

```diff
@@ -44,7 +45,7 @@ def async_to_blocking(coro):
 
 async def blocking_to_async(thread_pool, fun, *args, **kwargs):
     return await asyncio.get_event_loop().run_in_executor(
-        thread_pool, lambda: fun(*args, **kwargs))
+        thread_pool, functools.partial(fun, *args, **kwargs))
```

Comment:
you don't like my thunks? 😉

Reply:
The process pool won't work with them. This is what you have to do to get kwargs

Reply:
Heh. Unfortunate that lambdas are not pickle-able by default.